apt-hook/20apt-esm-hook.conf

```diff
@@ -1,5 +1,5 @@
 APT::Update::Pre-Invoke {
-	"[ ! -e /run/systemd/system ] || systemctl start --no-block apt-news.service || true";
+	"[ ! -e /run/systemd/system ] || systemctl start --no-block apt-news.service esm-cache.service || true";
```

Comment:
This one will never no-op because ESM Apps is for every LTS. It noops on non-LTS. ESM Infra is only set up on systems which need it, so we are not downloading any unneeded data here. I will add a comment to the service itself (as suggested in other comment in this review) explaining how it works for each service/release situatiuon.